parser_test.go

```diff
@@ -110,23 +111,35 @@ func TestParse(t *testing.T) {
 			continue
 		}
 
-		tags, err := Parse(testCase.filename, testCase.relative, basepath)
-		if err != nil {
-			t.Errorf("[%s] Parse error: %s", testCase.filename, err)
-			continue
-		}
+		// Repeat each test - once with all symbols and once without private symbols
+		for _, excludePrivate := range []bool{false, true} {
```

Comment:
Good point - was attempting to avoid redundancy but you are right - that it isn't a special case. Will change the tests.